test/acceptance_with_python/rbac/test_roles.py

```diff
@@ -19,10 +19,11 @@ def test_rbac_viewer_assign(
     # cannot delete with viewer permissions
     with pytest.raises(weaviate.exceptions.UnexpectedStatusCodeException) as e:
         viewer_client.collections.delete(name)
-
+    
     # with extra role that has those permissions it works
     with role_wrapper(admin_client, request, RBAC.permissions.config.delete(collection=name)):
-        viewer_client.collections.delete(name)
+        with pytest.raises(weaviate.exceptions.UnexpectedStatusCodeException) as e:
+            viewer_client.collections.delete(name)
```

Comment:
hmm why does this not work anymore? The viewer user is assigned the correct permissions to delete collections

Reply:
viewer role shouldn't be updated given it's a a builtin

Reply:
But we are not updating the viewer role, we are assigning to the viewer user.

Reply:
w8, i think i mixed it with something else,  isn't that trying to remove permission from the role `viewer_client.collections.delete(name)` ?